pkgs/applications/version-management/gita/default.nix

```diff
@@ -26,22 +26,8 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postUnpack = ''
-    for case in "\n" ""; do
-        substituteInPlace source/tests/test_main.py \
-         --replace "'gita$case'" "'source$case'"
-    done
-  '';
-
-  nativeCheckInputs = [
-    git
-    pytest
-  ];
-
-  checkPhase = ''
-    git init
-    pytest tests
-  '';
+  # 3 of the tests are failing
```

Comment:
Some more details would have been great

Reply:
Here is the case with the previous fix postUnpack: https://paste.gurkan.in/smooth-corgi.txt
Here is the case without: https://paste.gurkan.in/nice-wallaby.txt

(it failed to find the substitution which I assume is not valid for the new version, but nevertheless I didn't understand what was the fail reason exactlu